src/Lti/Extractor/ReplaceResultSourceIdExtractor.php

```diff
@@ -39,7 +39,7 @@ public function extractSourceId(string $xmlContent): int
             throw new InvalidLtiReplaceResultBodyException();
         }
 
-        $xml->registerXPathNamespace('x', 'http://www.imsglobal.org/lis/oms1p0/pox');
+        $xml->registerXPathNamespace('x', 'http://www.imsglobal.org/services/ltiv1p1/xsd/imsoms_v1p0');
```

Comment:
It seems that this is a breaking change. How can we be sure that TAO will send the correct namespace in LTI outcome call? What is the minimum TAO version we'll require from now on? Can you link a PR on TAO side where this namespace change was applied?

Reply:
https://github.com/oat-sa/extension-tao-ltideliveryprovider/pull/266
https://github.com/oat-sa/extension-tao-outcomelti/pull/50
https://github.com/oat-sa/simple-roster/pull/149
repo https://github.com/oat-sa/lti-director has been excluded, because it seems abandoned
The ticket: https://oat-sa.atlassian.net/browse/TAO-9835

Reply:
the matter is quite old ,  you can see inside the comments that Moodle has reported this in 2013

Reply:
Still, it could break a customer waiting for the old namespace, no? I think still is a breaking change.

Reply:
@Skyld is it enough the pr's provided or do we have to take into account some other vendors ?

Reply:
Sorry for the late reply @mike-ionut-mihai-sandu-tao . What I meant is that after this PR we'll have to specify a minimum sprint version on TAO side in every campaign where we use the roster to make sure we'll receive the correct namespace in LTI outcome call. Some client environments are pretty old, so we have to make sure we don't introduce a breaking change here. Could you please tell me which TAO community version should we use from now on as a minimum requirement? Thanks in advance.

Reply:
@Skyld because the merges on TAO are almost done for sure they will be captured inside the v0.137-alpha release tag, 
we have no strategy to backport this to a specific sprint and inside the ticket it is not requested 
so we can assume that the breaking changes will come with sprint 137

Reply:
I still have some concerns about this, this is a BC for basically all clients who will want to upgrade to this version but have TAO < v0.137 (basically all of them). What we should in my opinion is to expose the XML namespace through environment variable to make it configurable. The default value can be this new one, but it would also give the possibility to keep using the old one as well.

So what we need:

- Define a new environment variable (something like `LTI_OUTCOME_XML_NAMESPACE`)
- Make sure `ReplaceResultSourceIdExtractor` understands it
- Add meaningful default value for docker environment (`.env.docker`)
- Update tests (you can define test specific environment variables in `.env.test` file)
- Update devops documentation about the new variable: https://github.com/oat-sa/simple-roster/blob/develop/docs/devops/devops-documentation.md#lti-related-environment-variables
- Please target version 1.7.0 since this is not just a bugfix anymore

I am at your disposal if you need help. Thx in advance.

Reply:
done, i hope this is on your flavor , please take a look